src/client/components/Channels.tsx

```diff
@@ -57,6 +59,28 @@ export function ChannelButton({
   );
 }
 
+function DmButton({
```

Comment:
I like to think that the dm stands for dave miller

Reply:
"What's that?"  "Oh, that's the Dave Miller button"